fix(client): remove `outline: none` on Menu button

Sembauke opened the pull request:
Removed outline: none on  MenuButton so the outline becomes visible when pressing Tab.  Ref issue #40873.

Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

<!-- Feel free to add any additional description of changes below this line -->

Sembauke committed:
Removed outline: none on Menu button

Removed outline: none on  MenuButton so it becomes visible when pressing Tab.  Ref issue #40873.

raisedadead reviewed (commented):
I am not sure if this addresses the a11y issue entirely?

Sembauke commented:
> I am not sure if this addresses the a11y issue entirely?

I Don't understand what you mean?

ShaunSHamilton reviewed (changes requested):
I think we are looking more for:
```css
.toggle-button-nav:focus {
    outline: 5px auto -webkit-focus-ring-color !important;
    outline-offset: -3px;
}
```

As this will match the look of the Sign-in button.

---
I have removed the mention of this PR _closing_ the issue, as something still needs to be done (and we have not decided yet) about the certificate widget.

raisedadead commented:
@Sembauke if you get a chance can you update the PR with the recommend css changes mentioned above, as for the other issues we can leave it open for discussion.

Thanks for your work on this.

Sembauke commented:
> @Sembauke if you get a chance can you update the PR with the recommend css changes mentioned above, as for the other issues we can leave it open for discussion.
> 
> Thanks for your work on this.

No problem! I am new to GitHub and I am loving the kind support my first PR is getting

Sembauke committed:
Updated Menu button outline to match style of login button

This resolves the issue of not being able to focus on the menu button when pressing Tab.

naomi-lgbt reviewed (approved):
Looks good to me. Resolves the issue with keyboard focus.

As an aside, the outline appears when the dropdown is enabled - seems a bit odd, but I can live with it.

raisedadead commented:
#### :sparkles: :sparkles: **AWESOME! LGTM.** :sparkles: :sparkles: 

Thanks @Sembauke, for this pull request and contributing to the code-base for the first time.

We will be looking forward for more contributions in future.

Cheers and happy contributing!
